routersploit/modules/libs/lzs.py

```diff
@@ -0,0 +1,140 @@
+# !/usr/bin/env python
```

Comment:
I have transformed `utils` module to python package. You can add `lzs` as a module within `utils` package e.g. `routersploit.utils.lzs`. Now we can start thinking about factoring out few things from bloated `utils`